packages/sit-onyx/src/components/OnyxToast/OnyxToast.vue

```diff
@@ -0,0 +1,253 @@
+<script lang="ts" setup>
+import circleAttention from "@sit-onyx/icons/circle-attention.svg?raw";
+import circleCheck from "@sit-onyx/icons/circle-check.svg?raw";
+import circleInformation from "@sit-onyx/icons/circle-information.svg?raw";
+import circleX from "@sit-onyx/icons/circle-x.svg?raw";
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import { computed } from "vue";
+import { useDensity } from "../../composables/density";
+import { injectI18n } from "../../i18n";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import type { OnyxToastProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxToastProps>(), {
+  color: "neutral",
+  duration: 5000,
+  clickable: false,
+  icon: undefined, // needed to prevent default value being "false"
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the toast is clicked. Requires `clickable` property to be enabled.
+   */
+  click: [];
+  /**
+   * Emitted when the toast should be closed.
+   */
+  close: [];
+}>();
+
+const { t } = injectI18n();
+const { densityClass } = useDensity(props);
+
+const hasProgressBar = computed(() => props.duration > 0);
+
+const DEFAULT_ICONS: Record<typeof props.color, string> = {
+  neutral: circleInformation,
+  danger: circleX,
+  warning: circleAttention,
+  success: circleCheck,
+};
+
+const icon = computed(() => {
+  if (props.icon === false) return;
+  return props.icon || DEFAULT_ICONS[props.color];
+});
+</script>
+
+<template>
+  <component
+    :is="props.clickable ? 'button' : 'div'"
+    class="onyx-toast"
+    :class="[`onyx-toast--${props.color}`, densityClass]"
+    role="alert"
+    :aria-label="props.clickable ? props.headline : undefined"
+    @click="props.clickable && emit('click')"
+  >
+    <div class="onyx-toast__wrapper">
+      <OnyxIcon v-if="icon" :icon="icon" />
+
+      <div class="onyx-toast__content onyx-truncation-ellipsis">
+        <div class="onyx-toast__headline onyx-text">
+          <span class="onyx-truncation-ellipsis"> {{ props.headline }}</span>
+
+          <button
+            v-if="!props.clickable"
+            :aria-label="t('close')"
+            class="onyx-toast__close"
+            @click="emit('close')"
+          >
+            <OnyxIcon :icon="xSmall" />
+          </button>
+        </div>
+
+        <p
+          v-if="props.description"
+          class="onyx-toast__description onyx-text--small onyx-truncation-multiline"
+          tabindex="0"
```

Comment:
Why does the description need to be tabbable? Only interactive elements should be tabbable.
```suggestion
```

Reply:
Because axe tests failed because the description gets scrollable after 3 lines and scrollable elements must be focussable

Reply:
Good point! Then we should also pause the timer on focus-within and also show the focus-visible effect on focus-within